browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -15338,7 +15330,7 @@ index 0000000000000000000000000000000000000000..d9f566a6cce6433c2cfbb25fd6793f65
 +        items.append(language);
 +    }
 +
-+    browserContext->processPool->configuration().setOverrideLanguages(WTFMove(items));
++    WebKit::setOverrideLanguages(WTFMove(items));
```

Comment:
This is going to break our tests, we need language override to be per context so that different tests with different language settings can run in parallel in a browser.

Reply:
I dropped the API workaround and instead added the bare minimum revert of the changes that broke it: https://github.com/q66/WebKit/commit/ec4c7e874e004be2fffcacdca77d36dc54304e6b

It's not an exact revert, as that would make the patch grow quite a bit more (there are at least two commits and they are both bigger than this). Instead, I picked the changes from both while taking care of having to move around as little as possible.

I think we could have this for now, and drop those patch bits once the situation is solved upstream? I marked them with comments to make it clear what will need dropping.